src/Backend/Modules/Tags/Installer/Data/install.sql

```diff
@@ -1,8 +1,15 @@
 CREATE TABLE IF NOT EXISTS `tags` (
```

Comment:
this file should be completely removed in favour of installing doctrine entities (see https://github.com/forkcms/forkcms/blob/doctrine/src/Backend/Modules/ContentBlocks/Installer/Installer.php#L34-L36)